### PR TITLE
Bug/fix bugs in users userid function + redis

### LINKED ISSUE #212: [Bug] HttpTriggerAPIUsersId "get" method undefined variables
**Describe the bug**

The get method on line 66 has many undefined variables. It uses userId, but it is not an arg/param. It also uses variables columns and data.

Screen Shot 2021-02-26 at 6 38 47 PM
Screen Shot 2021-02-26 at 6 38 56 PM
Screen Shot 2021-02-26 at 6 39 10 PM

### PR DESCRIPTION
**Pull request by farhadbahrehmandhenry**
the most straightforward way to test this is to go to this [URL](https://nscstrdevusw2thucommon.z5.web.core.windows.net/) and test the endpoints. or use that URL and user postman to test the different method.

### CODE REVIEW

File: functions/HttpTriggerAPIUsersId/__init__.py [hunk 1]
```diff
@@ -5,189 +5,259 @@
...
-        logging.info("Caching complete!")
-    except TypeError as e:
-        logging.info("Caching failed")
-        logging.info(e.args[0])
+        logging.debug('User was updated successfully!.')
+
+        return func.HttpResponse('User updated', status_code=200)
+
+
+def patch_user(user_req_body, conn, user_id):
```
Comment (theKunte) [inline functions/HttpTriggerAPIUsersId/__init__.py]: Patch method at the moment does not update the user correctly. It updates it but not with the correct data

File: functions/HttpTriggerAPIUsersId/__init__.py [hunk 2]
```diff
@@ -5,189 +5,259 @@
...
+    logging.debug('''
+      Going to execute PATCH query on user {}
+    '''.format(user_id))
+
+    fieldsToUpdate = list(user_req_body.keys())
+    updatableFields = ['firstName', 'lastNAme', 'emeil']
```
Comment (theKunte) [inline functions/HttpTriggerAPIUsersId/__init__.py]: Some typos on line 201 ( lastNAme and emeil)

**Review by theKunte — APPROVED** [review]
Since I was not able to test the latest changes locally, Farhad showed me the changes he made for the HTTPTriggerAPIUsersUserId. Methods are now working. One thing that will be addressed later is the cache. Everything else looks good. Good work Farhad and thanks for the extra effort!

**Comment by theKunte** [comment]
Add some more description to this PR so other people know how to test it @chonger878  this PR fixes the UsersUserId endpoint. Have a look and try it out locally.

**Review by Kemet2016 — APPROVED** [review]
I tested it and it works.

**Pull request merged**